### ISSUE TITLE
Getting error if path has arc

### ISSUE THREAD

**Issue by ranaraumik16** [dy/svg-path-bounds#5]
I am trying to get bounds of svg path data
"M 10 10 A 10 10 0 0 0 50 50 A 10 10 0 0 0 10 10 z"

But getting error
Uncaught TypeError: arcToCurve is not a function
at normalize (index.js:30:1)
at pathBounds (index.js:24:1)

**Comment by dy** [comment]
are you sure all dependencies are installed?

**Comment by ranaraumik16** [comment]
I have run command 
```npm i svg-path-bounds```

Do I need to install another modules separately?

**Comment by ranaraumik16** [comment]
Upon investigation, I discovered an issue. Specifically, I attempted to create a basic Node.js application, and it performed well. However, when I attempted to develop a new React application and utilized the "svg-path-bounds" library, I encountered an error related to the "normalize-svg-path" package.
![image](https://github.com/dy/svg-path-bounds/assets/65937635/b3343987-217b-4179-bc15-e1c6ceedf394)
